Source/WebCore/page/Page.h

```diff
@@ -273,6 +273,7 @@ enum class RenderingUpdateStep : uint32_t {
 #endif
     RestoreScrollPositionAndViewState = 1 << 23,
     AdjustVisibility                  = 1 << 24,
+    Reveal                            = 1 << 25,
```

Comment:
Would be good to have this reflect the actual order

Reply:
I didn't do this one, seems like it's not the case for lots of others either. I can if you feel strongly :)

Reply:
It is slightly confusing to see this but later realize it's actually one of the first steps. I think Simon likes to keep this in sync with the actual order too (although arguably we failed to do that over the years)